### PR TITLE
Ammo / gun vendors

### PR DESCRIPTION
**Pull request by X0-11**
see #259 for details
Map Changes:
0bfba5ae85b64e3e89f9d288a7be28e5
Small update to medical's stairway
1023bc028bb14965af297bb84246f18b

### CODE REVIEW

**Comment by X0-11** [comment]
Okay. So no map editing so far. It merges.

**Comment by X0-11** [comment]
There we go, this now has vendors replacing the lockers in the map. And no conflicts. Finally.

**Comment by BDpuffy420** [comment]
I'm digging it!

File: maps/unsc_frigate-3.dmm [hunk 1]
```diff
@@ -297,7 +297,7 @@
...
 "fL" = (/obj/effect/wingrille_spawn/reinforced,/turf/simulated/floor/tiled/white,/area/unscfrigate/medbay/exam)
 "fM" = (/obj/structure/closet/secure_closet/medical_wall{name = "Pill Cabinet"; pixel_x = -32; pixel_y = 0},/obj/item/weapon/reagent_containers/syringe/antiviral,/obj/item/weapon/storage/pill_bottle/antitox,/obj/item/weapon/storage/pill_bottle/tramadol,/obj/structure/table/standard{name = "plastic table frame"},/obj/item/weapon/reagent_containers/syringe/ld50_syringe/triadrenaline,/turf/simulated/floor/tiled/white,/area/unscfrigate/medbay/exam)
-"fN" = (/turf/simulated/floor/tiled,/area/unscfrigate/medbay/exam)
+"fN" = (/obj/machinery/body_scanconsole{tag = "icon-body_scannerconsole (WEST)"; icon_state = "body_scannerconsole"; dir = 8},/turf/simulated/floor/tiled/white,/area/unscfrigate/medbay/exam)
```
Comment (bloxgate) [inline maps/unsc_frigate-3.dmm:300]: Was something changed in the medical area too? I didn't see anything about it in the PR description.

**Comment by X0-11** [comment]
Ah. Sorry! It was a small fix that I forgot in my previous mapping PR. Let me upload a picture.

**Review by bloxgate — APPROVED** [review]
LGTM

File: maps/unsc_frigate-3.dmm (hunk 1 above)
Comment (caelaislinn) [reply maps/unsc_frigate-3.dmm:300]: Apparently we fixed the issue with the misaligned stairs here because I can't find it any more

**Comment by bloxgate** [comment]
Yeah, medical needs a lot of work at the moment. I'm tempted to redo a lot of that area of the deck myself.

**Review by caelaislinn — APPROVED** [review]

**Pull request merged**